### PR TITLE
Homepage revamp!

### LINKED ISSUE #4660: [www] Update website content - Home and Pricing
Update the Home and Pricing pages to reflect new messaging.
The Invision link below is peppered with notes that will guide you in building out the pages. Let me know if you have any questions.

**PREVIEW - Clickable prototype**
https://chef.invisionapp.com/share/ZVG7KTWK267#/283100669_step_1

**PREVIEW - Home and Enterprise mobile mockups --> including nav toggle**
https://chef.invisionapp.com/share/ZVG7KTWK267#/287144929_infographic_Mobile_View__-_V3_Step_1

-------------

**GET STYLES & COPY - Home page Invision Inspect**
https://chef.invisionapp.com/d/main/#/console/13570590/283100669/inspect

**GET STYLES & COPY - Pricing page Invision Inspect**
https://chef.invisionapp.com/d/main/#/console/13570590/286280609/inspect

**GET STYLES & COPY - Homepage mobile (with infographic assets) Invision Inspect**
https://chef.invisionapp.com/d/main/#/console/13570590/287144929/inspect (circles as a single asset)
https://chef.invisionapp.com/d/main/#/console/13570590/287144927/inspect (circles as separate 5 assets)

**New assets (icons, habicats, etc.)**
https://www.dropbox.com/s/2q1rhem44vmqdch/Habitat-v3-assets.zip?dl=0
You can also download assets from Invision Inspect either by clicking on the layer or switching over to the ASSETS tab:
screenshot 2018-03-21 10 04 22

### PR DESCRIPTION
**Pull request by mgamini**
tenor-168728392

Signed-off-by: Garrett Amini <kadjar@gmail.com>

Fixes #4660.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by cnunciato** [comment]
@mgamini One thing I'm noticing is that when I'm scrolled, if I resize, I lose the nav:

https://d3vv6lp55qjaqc.cloudfront.net/items/203a3u423O290w0n451R/Screen%20Recording%202018-04-05%20at%2012.01%20PM.gif?X-CloudApp-Visitor-Id=c7478a37cdf244e2b3b9b650ba7da0fa&v=8efc8589

**Comment by cnunciato** [comment]
I also see`li` bullets between the carousel indicators (in FF, at least):

image

**Comment by cnunciato** [comment]
These should be left-justified:

image

**Comment by cnunciato** [comment]
On the tutorial pages (linked from home), I'm seeing this for the nav:

image

**Comment by cnunciato** [comment]
These buttons should be the same height:

image

**Comment by cnunciato** [comment]
At tablet width and below, the carousel gets weird, and can cause side scrolling:

image

**Comment by cnunciato** [comment]
There's a one-pixel gap on the top and bottom of the video (as though it's not quite sized right):

image

**Comment by cnunciato** [comment]
At desktop widths below 1200px, we get side-scrolling:

**Comment by cnunciato** [comment]
The mocks have the calls to action as lowercase, and the spacing around the symbols is much tighter:

**Comment by cnunciato** [comment]
This is a minor quibble, but the dissolve between carousel "slides" feels a bit too long. Maybe try reducing the transition duration by 1/3 or 1/2.

File: www/source/javascripts/nav.js [hunk 1]
```diff
@@ -104,6 +104,7 @@ var toggleStickyNav = function() {
 toggleStickyNav();
 
 $navToggle.click(function() {
+  console.log('nav')
```
Comment (cnunciato) [inline www/source/javascripts/nav.js]: Can drop this.

File: www/source/stylesheets/_carousel.scss [hunk 2]
```diff
@@ -15,4 +15,182 @@ $slider-transition: 1s;
...
+    &.arrow-right {
+      background: url('/images/icons/icon-chevron-right.svg') center center no-repeat transparent;
+    }
+  }
+}
+
+.carousel-nav-dots {
+  $carousel-nav-dot-color: #c3c6c8;
+  display: flex;
+  // align-content: center;
```
Comment (cnunciato) [inline www/source/stylesheets/_carousel.scss]: Remove commented lines.

File: www/source/stylesheets/_nav.scss [hunk 3]
```diff
@@ -56,9 +57,9 @@ $main-nav-breakpoint: 769px;
 
   // modifiers for nav on homepage on large screens
   .home &:not(.is-sticky) {
-    @include large-nav {
-        background: transparent;
-    }
+    // @include large-nav {
```
Comment (cnunciato) [inline www/source/stylesheets/_nav.scss]: Remove commented lines.

**Comment by ryankeairns** [comment]
@cnunciato I changed the link styles to be more like other Chef branding, thus the caps and such. As for the logos, they shouldn't look like that - i.e. they should match the mock. I had it styled that way, perhaps it got munged (or I messed it up whilst working on mobile tweaks).

**Comment by mgamini** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362858038).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/362858038).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by mgamini** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362864833).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/362864833).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by mgamini** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362882311).

**Comment by mgamini** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362885412).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/362885412).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362913244).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/362913244).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by cnunciato** [comment]
@thesentinels acceptance

with feeling

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362931649).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/362931649).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by mgamini** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/362940520).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/362940520).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**